src/Cli/dotnet/commands/dotnet-workload/search/WorkloadSearchCommand.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CommandLine.Parsing;
+using System.IO;
+using Microsoft.Deployment.DotNet.Releases;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.Utils;
+using Product = Microsoft.DotNet.Cli.Utils.Product;
+using Microsoft.NET.Sdk.WorkloadManifestReader;
+using System.Linq;
+
+namespace Microsoft.DotNet.Workloads.Workload.Search
+{
+    internal class WorkloadSearchCommand : CommandBase
+    {
+        private readonly IReporter _reporter;
+        private readonly VerbosityOptions _verbosity;
+        private readonly IWorkloadResolver _workloadResolver;
+        private readonly ReleaseVersion _sdkVersion;
+        private readonly string _workloadIdStub;
```

Comment:
is id stub also a name for "search term"?

Reply:
Yes, that's what I intended.